src/averageAges.js

```diff
@@ -14,12 +14,19 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const manAges = people.filter((person) => {
+    return (century !== undefined
+      ? person.sex === 'm' && (Math.ceil(person.died / 100)) === century
+      : person.sex === 'm');
```

Comment:
```suggestion
    return century
      ? person.sex === 'm' && (Math.ceil(person.died / 100)) === century
      : person.sex === 'm';
```
fix comparing with undefined everywhere